operations/template/alert.tf

```diff
@@ -217,6 +217,88 @@ resource "azurerm_monitor_metric_alert" "azure_5XX_alert" {
   }
 }
 
+resource "azurerm_monitor_metric_alert" "ti_dynamic_memory_alert" {
```

Comment:
Does ti need to be included in the resource, "ti_dynamic_memory_alert", if it will be in the name, "cdcti-${var.environment}-dynamic-memory-alert", especially as it's in the TI repo already?